android/build.gradle

```diff
@@ -368,16 +366,56 @@ task createNativeDepsDirectories(dependsOn: applyJavaPatches) {
     thirdPartyNdkDir.mkdirs()
 }
 
-task downloadBoost(dependsOn: createNativeDepsDirectories, type: Download) {
+def resolveTaskFactory(String name, String artifactLocalName, File reactNativeAndroidDownloadDir, File reanimatedDownloadDir) {
+    return tasks.create(name: name.startsWith("resolve") ? name : "resolve" + name, dependsOn: createNativeDepsDirectories, type: Copy) {
```

Comment:
Why `name.startsWith("resolve") ? name : "resolve" + name` instead of just `name`?

Reply:
[86ce034](https://github.com/software-mansion/react-native-reanimated/pull/3588/commits/86ce034d14d8e16e0c8de6f353144e6921b29802)